src/tools/android/java/com/google/devtools/build/android/AndroidResourceOutputs.java

```diff
@@ -59,8 +59,8 @@
     private static final long MINIMUM_TIMESTAMP_INCREMENT = 2000L;
 
     // The earliest date representable in a zip file, 1-1-1980 (the DOS epoch).
```

Comment:
let's update this comment with the new date, and why it was chosen

Reply:
(ping @benjaminRomano)

Reply:
Sorry for delay, I updated the comment with some more context.